### PR TITLE
feat: create-tabs-component

### PR DESCRIPTION
**Pull request by brendapc**
Create tabs view component with mocked data
image
image

### CODE REVIEW

**Comment by LucasSusin** [comment]
Can you remove all the files and code not related to this US? And resolve the conflicts

**Comment by brendapc** [comment]
> Can you remove all the files and code not related to this US? And resolve the conflicts
@LucasSusin fixed

File: src/styles/colors.ts [hunk 1]
```diff
@@ -13,3 +13,5 @@ export const link = "#141bc3"
 export const defaultMessage = "#1450c31a"
 export const errorMessage = "#e57878"
 export const successMessage = "#b6eaa1"
+
+export const mediumGray = "#2C2C2C"
```
Comment (luizmellodev) [inline src/styles/colors.ts]: Can you move this to line 6, with the other gray's?

File: src/components/TabsView/index.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+              }
+              label={pdf.title}
+              key={pdf.title}
+            />
+          ))}
+        </Tabs>
+      </TabsWrapper>
+      {pdfList.map((pdf: any) => (
+        <TabPanel value={value + 1} index={pdf.id} key={pdf.title}>
+          {pdf.id}
```
Comment (luizmellodev) [inline src/components/TabsView/index.tsx:68]: You can use the index here. I don't think the pdf list will come with an id

File: src/components/TabsView/index.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
+import * as React from "react"
```
Comment (luizmellodev) [inline src/components/TabsView/index.tsx]: No need to import React anymore

File: src/components/TabsView/index.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        <Box sx={{ p: 3 }}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  )
+}
+
+export default function TabsView({ pdfList }: any) {
+  const [value, setValue] = React.useState(0)
```
Comment (luizmellodev) [inline src/components/TabsView/index.tsx]: You can import useState only for React
```suggestion
import { useState } from "React"
  const [value, setValue] = useState<number>(0)
```

**Review by luizmellodev — COMMENTED** [review]
Isn't the "div" where the inputs will be missing?
Trello: Descrição: Criar a estrutura/componente do campo de visualização (isso inclui o sistema de troca de abas de veículos)
 
Captura de Tela 2023-05-22 às 18 33 59

File: src/components/TabsView/index.tsx (hunk 2 above)
Comment (brendapc) [reply src/components/TabsView/index.tsx:68]: this is just to illustrate the change of tab, it shouldn't show up on the final page

**Pull request merged**